docs/en/for-developers/installing-bhima.md

```diff
@@ -51,6 +51,23 @@ sudo apt-get update && sudo apt-get install yarn --no-install-recommends
 sudo apt-get install git
 ```
 
+### Getting dependencies for Puppeteer
+
+By default, your OS might not have all dependencies for running puppeteer, the PDF renderer based based on Chromium.  These vary by OS.  For Ubuntu/Debian, you can run the following to prepare your environment:
+
+```bash
+# Install puppeteer dependencies on Ubuntu
+sudo apt-get install ca-certificates fonts-liberation gconf-service \
+  libappindicator1 libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2  \
+  libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 \
+  libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 \
+  libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
+  libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 \
+  libxss1 libxtst6 lsb-release wget -y
```

Comment:
👍 hahaha

Reply:
🤷

Reply:
There are some issues on linux if we do not install all these staff